app/views/orders/show.xml.erb

```diff
@@ -63,9 +63,7 @@
       <!--Scheduled Station AE Title-->
       <attr tag="00400001" vr="AE"><%= order.scheduled_station_ae_title %></attr>
       <!--Scheduled Procedure Step Start Date-->
-      <attr tag="00400002" vr="DA"><%= order.scheduled_procedure_step_start_datetime.split("T")[0].gsub("-","")  if order.scheduled_procedure_step_start_datetime.present? %></attr>
-      <!--Scheduled Procedure Step Start Time-->
-      <attr tag="00400003" vr="TM"><%= order.scheduled_procedure_step_start_datetime.split("T")[1].gsub(":","")  if order.scheduled_procedure_step_start_datetime.present? %></attr>
+      <attr tag="00404005" vr="DT"><%= order.scheduled_procedure_step_start_datetime.gsub("-","").gsub("T","").gsub(":","") + ".000000"  if order.scheduled_procedure_step_start_datetime.present? %></attr>
```

Comment:
So, this is basically change from a time only tag to a date-time tag.

Format must be `YYYYMMDDHHMMSS.FFFFFF&ZZXX`
https://dicom.nema.org/medical/dicom/2017a/output/chtml/part05/sect_6.2.html

As long as `order.scheduled_procedure_step_start_datetime` doesn't end in a `Z` or time offset, this looks like it will work.

Reply:
Yes.  We are not sending a Z to mowoli, so we should be good not including "&ZZXX", unless of course BFLY needs it.  I'm a little frustrated the BFLY engineer couldn't tell us what BFLY needs and just told us to try stuff.  :-(